package.json

```diff
@@ -75,6 +75,7 @@
     "express": "^4.18.2",
     "knex": "^2.4.2",
     "multer": "^1.4.5-lts.1",
+    "node-fetch": "^3.3.1",
```

Comment:
now we're on node 18, `fetch` is built-in, you don't need the package
https://dev.to/cloudx/nodejs-18-fetch-api-test-runner-module-and-more-2ckg

Reply:
I kept getting a depcheck error that it was missing, even though it wasn't and I removed `node_modules` and ran `npm i` and still got the error

Reply:
you import it `import fetch from 'node-fetch'` in `ipfsStatus.ts`. Can delete that

Reply:
`fetch` works without an import in the file